sonar-csharp-plugin/pom.xml

```diff
@@ -215,6 +216,16 @@
                       </fileset>
                     </copy>
 
+                    <!-- These files are only used are only used by old scanners. Their only function is to produce a good error message and faild the build. -->
+                    <copy file="${project.build.directory}/../../sonaranalyzer-dotnet/src/SonarAnalyzer.DummyScanner/bin/${analyzer.configuration}/DummyScanner.exe"
+                          tofile="${sonarAnalyzer.workDirectory}/MSBuild.Runner/SonarQube.MSBuild.PreProcessor.exe" />
+
```

Comment:
I would remove the spaces to make it clear these tasks belong together. I don't whether there is any guideline on that I just know it is what I did everywhere else on this file.